app/v6/lossClaims/delete/DeleteLossClaimService.scala

```diff
@@ -35,6 +35,7 @@ class DeleteLossClaimService @Inject() (connector: DeleteLossClaimConnector) ext
 
   private val errorMap: Map[String, MtdError] = Map(
     "INVALID_TAXABLE_ENTITY_ID" -> NinoFormatError,
+    "INVALID_TAX_YEAR"          -> TaxYearFormatError,
```

Comment:
This should be mapped to [TaxYearClaimedForFormatError](https://github.com/hmrc/individual-losses-api/blob/main/app/common/errors/LossesApiCommonErrors.scala#L29). Please update, thanks